02-Logging/01-Logging-to-a-GSD-file.ipynb

```diff
@@ -48,7 +48,11 @@
   {
```

Comment:
<p>Use <code>rm -f</code>  so that it is not an error when the file is missing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>